tasks/clean_previous_cloudformation.yml

```diff
@@ -0,0 +1,37 @@
+---
+
+- set_fact:
+    previous_cf: "{{ item.tags | selectattr('key', 'equalto', 'Name') | map(attribute='value') | join(',')}}"
+
+- cloudformation_facts:
+    profile: "{{ boto_profile | default(omit) }}"
+    region: "{{ wimpy_aws_region }}"
+    stack_name: "{{ previous_cf }}"
+    all_facts: true
+  register: cf_facts
+
+- name: "Delete ELB from DNS"
+  route53:
+    profile: "{{ boto_profile | default(omit) }}"
+    region: "{{ wimpy_aws_region }}"
+    command: create
+    overwrite: True
+    zone: "{{ wimpy_aws_hosted_zone_name }}"
+    record: "{{ wimpy_aws_dns_name }}.{{ wimpy_aws_hosted_zone_name }}."
+    identifier: "{{ previous_cf }}"
+    weight: "0"
```

Comment:
This removes the CNAME of the old ELB from the DNS, so clients resolving the project's DNS will receive only the new ELB. Clients that had already resolved the DNS and are using the old ELB will re-resolve the DNS when the TTL is reached, so we wait for $TTL * 2 seconds before removing the old CloudFormation.